scripts/webservice-image-digest.py

```diff
@@ -20,14 +20,19 @@
 
 DOCKER_TAG_BASE = "quay.io/dockstore/dockstore-webservice"
 
+def get_commit_from_tag_url(tag_url):
+  # takes a tag url and gets the commit hash it is pointed at
```

Comment:
Will this work for non-annotated tags? We typically won't have them, as Hubflow always creates an annotated tag, but there might be a test case or something where we try with a lightweight tag.

Reply:
Added another commit to handle simple tags